### ISSUE TITLE
More testing of ceph driver in BAT

### ISSUE THREAD

**Issue by rbradford** [01org/ciao#842]
Now we have a ceph test as part of the BAT we should make sure that as much of ciao-storage/ceph.go is covered by this test.

**Comment by kaccardi** [comment]
@markdryan we need a way to understand what percentage of ciao-storage is covered by bat.

**Comment by markdryan** [comment]
Agreed.  We should do this for all our BAT tests.  There's no easy way to do this but I found a blog  https://blog.cloudflare.com/go-coverage-with-external-tests/ that describes a workaround.   We should be able to get this to work.
